ci/scripts/pr_list_database.py

```diff
@@ -114,11 +119,12 @@ def input_args():
     parser = ArgumentParser(description=description,
                             formatter_class=ArgumentDefaultsHelpFormatter)
 
-    parser.add_argument('sbfile', help='SQLite3 database file with PR list', type=str)
+    parser.add_argument('--sbfile', help='SQLite3 database file with PR list')
```

Comment:
Why was the type removed?

Reply:
"type removed"?? I added this switch to the argument list because another argument list was not fixed.

Reply:
Oh wait, I missed the wrap around, yes the type was changed because it is being passed as a list now.  You can see the list is applied to the database in a loop depending the number of arguments.  This may need work and we have decided to improve on this interface for another PR.

Reply:
I just went through the PR and I do not see anywhere where `--sbfile` is passed multiple files. Further, I do not see a use case where we would want to use multiple _databases_.

Also, if you wanted to have a list, shouldn't you would use `nargs`?:
```suggestion
    parser.add_argument('--sbfiles', help='SQLite3 database file with PR list', nargs='+', type=str)
```

Reply:
Oh, I misspoke,  `sbfiles` is no longer positional because `update_pr` takes a variable number of arguments and there the type change here was for another reason.

Reply:
BTW, the type was removed, not changed, because it had to take on a path and there was an issue with it being a string, not sure why that matters.

Reply:
I'm not sure why `str` wouldn't work anymore, but I'll take your word for it. Leave this unresolved in case Rahul wants to chime in.

Reply:
me neither, let me check ...

`(gfs_workflow) mterry (Orion-login-1) scripts (bug_1649_Ready-Restart) $ git diff pr_list_database.py 
diff --git a/ci/scripts/pr_list_database.py b/ci/scripts/pr_list_database.py
index 8f519e0f..a099ff51 100755
--- a/ci/scripts/pr_list_database.py
+++ b/ci/scripts/pr_list_database.py
@@ -122,7 +122,7 @@ def input_args():
     parser = ArgumentParser(description=description,
                             formatter_class=ArgumentDefaultsHelpFormatter)
 
-    parser.add_argument('--sbfile', help='SQLite3 database file with PR list')
+    parser.add_argument('--sbfile', help='SQLite3 database file with PR list', type=str)
     parser.add_argument('--create', help='create sqlite file for pr list status', action='store_true', required=False)
     parser.add_argument('--add_pr', nargs=1, metavar='PR', help='add new pr to list (defults to: Open,Ready)', required=False)
     parser.add_argument('--remove_pr', nargs=1, metavar='PR', help='removes pr from list', required=False)
(gfs_workflow) mterry (Orion-login-1) scripts (bug_1649_Ready-Restart) $ ./pr_list_database.py --display --sbfile /work2/noaa/stmp/GFS_CI_ROOT/open_pr_list.db
1601 Open Built 0`

Yep, no big deal here.

Reply:
Me neither, here let me check ...

Reply:
```
(gfs_workflow) mterry (Orion-login-1) scripts (bug_1649_Ready-Restart) $ git diff pr_list_database.py 
diff --git a/ci/scripts/pr_list_database.py b/ci/scripts/pr_list_database.py
index 8f519e0f..a099ff51 100755
--- a/ci/scripts/pr_list_database.py
+++ b/ci/scripts/pr_list_database.py
@@ -122,7 +122,7 @@ def input_args():
     parser = ArgumentParser(description=description,
                             formatter_class=ArgumentDefaultsHelpFormatter)
 
-    parser.add_argument('--sbfile', help='SQLite3 database file with PR list')
+    parser.add_argument('--sbfile', help='SQLite3 database file with PR list', type=str)
     parser.add_argument('--create', help='create sqlite file for pr list status', action='store_true', required=False)
     parser.add_argument('--add_pr', nargs=1, metavar='PR', help='add new pr to list (defults to: Open,Ready)', required=False)
     parser.add_argument('--remove_pr', nargs=1, metavar='PR', help='removes pr from list', required=False)
(gfs_workflow) mterry (Orion-login-1) scripts (bug_1649_Ready-Restart) $ ./pr_list_database.py --display --sbfile /work2/noaa/stmp/GFS_CI_ROOT/open_pr_list.db
1601 Open Built 0
```

Reply:
Yep, no big deal.